### ISSUE TITLE
duplicated time index supporting on `getindex`

### ISSUE THREAD

**Issue by iblis17** [JuliaStats/TimeSeries.jl#458]
Since we starting to move toward accepting dup time index, `cl[[Date(2001, 10, 1), Date(2001, 5, 1), Date(2011, 5, 1)]]` should output with duplicated time index. 

Current output:
```julia
julia> cl[[Date(2001, 10, 1), Date(2001, 5, 1), Date(2011, 5, 1)]]
2×1 TimeArray{Float64,1,Date,Array{Float64,1}} 2001-05-01 to 2001-10-01
│            │ Close │
├────────────┼───────┤
│ 2001-05-01 │ 25.93 │
│ 2001-10-01 │ 15.54 │

```

**Comment by sencilla** [comment]
So when is this feature available?

**Comment by iblis17** [comment]
Hi @sencilla 
The patch is included in v0.19.x https://github.com/JuliaStats/TimeSeries.jl/commit/ed0cd207d379fe7959a00e5772034fcaec4b5eb5
